tools/envoy_build_fixer.py

```diff
@@ -5,16 +5,46 @@
 import sys
 
 LICENSE_STRING = 'licenses(["notice"])  # Apache 2\n'
+ENVOY_PACKAGE_STRING = (
+    'load("//bazel:envoy_build_system.bzl", "envoy_package")\n'
+    '\n'
+    'envoy_package()\n')
+
 
 def FixBuild(path):
   with open(path, 'r') as f:
     outlines = [LICENSE_STRING]
+    load = 0
+    seen_ebs = 0
+    seen_epkg = 0
     for line in f:
+      if line.startswith('package(') and not path.endswith(
+          'bazel/BUILD') and not path.endswith('ci/prebuilt/BUILD'):
+        continue
+      if load == 0 and line.startswith('load('):
+        load = 1
+      if load == 1:
+        if 'envoy_build_system.bzl' in line:
+          seen_ebs = 1
+        if 'envoy_package' in line:
+          seen_epkg = 1
+        if line.strip().endswith(')'):
+          load = 2
+          if seen_ebs:
+            if not seen_epkg:
+              outlines.append(line.strip()[:-1] + ', "envoy_package")\n')
```

Comment:
Good point, fixed (although it doesn't really matter here, since it's either going to be one liner or closing bracket will be at the first column anyway).